test/JDBC/expected/parallel_query/with_recompile-vu-verify.out

```diff
@@ -0,0 +1,2991 @@
+              Index Cond: (c71 = 0)
+  Query Text: select * from tb_recomp_71 where c71 = 0
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Index Scan using tb_recomp_71_pkey on tb_recomp_71
+              Index Cond: (c71 = 0)
+~~END~~
+
+set babelfish_showplan_all off
+go
+SELECT current_setting('plan_cache_mode') as plan_mode_after_call
+go
+~~START~~
+text
+auto
+~~END~~
+
+exec    p_recomp_71 0  -- sets plan_cache_mode=force_generic_plan
+go
+~~START~~
+int#!#int
+~~END~~
+
+~~START~~
+int#!#int
+~~END~~
+
+~~START~~
+text
+force_generic_plan
+~~END~~
+
+~~START~~
+int#!#int
+~~END~~
+
+~~START~~
+int#!#int
+~~END~~
+
+SELECT current_setting('plan_cache_mode') as plan_mode_after_call
+go
+~~START~~
+text
+force_generic_plan
+~~END~~
+
+
+select set_config('plan_cache_mode', 'auto', false)
+go
+~~START~~
+text
+auto
+~~END~~
+
+set babelfish_showplan_all on
+go
+exec    p_recomp_71 0 with recompile    -- sets plan_cache_mode=force_generic_plan
+go
+~~START~~
+text
+Query Text: EXEC    p_recomp_71 0 /* EXECUTE WITH RECOMPILE */
+  Query Text: select * from tb_recomp_71 where c71 = "@p"
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Index Scan using tb_recomp_71_pkey on tb_recomp_71
+              Index Cond: (c71 = 0)
+  Query Text: select * from tb_recomp_71 where c71 = 0
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Index Scan using tb_recomp_71_pkey on tb_recomp_71
+              Index Cond: (c71 = 0)
+  Query Text: select set_config('plan_cache_mode', 'force_generic_plan', false)
+  ->  Result
+  Query Text: select * from tb_recomp_71 where c71 = "@p"
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Index Scan using tb_recomp_71_pkey on tb_recomp_71
+              Index Cond: (c71 = 0)
+  Query Text: select * from tb_recomp_71 where c71 = 0
+  ->  Gather
+        Workers Planned: 1
+        Single Copy: true
+        ->  Index Scan using tb_recomp_71_pkey on tb_recomp_71
+              Index Cond: (c71 = 0)
+~~END~~
+
+set babelfish_showplan_all off
+go
+SELECT current_setting('plan_cache_mode') as plan_mode_after_call
+go
+~~START~~
+text
+auto
+~~END~~
+
+exec    p_recomp_71 0 with recompile -- sets plan_cache_mode=force_generic_plan
+go
+~~START~~
+int#!#int
+~~END~~
+
+~~START~~
+int#!#int
+~~END~~
+
+~~START~~
+text
+force_generic_plan
+~~END~~
+
+~~START~~
+int#!#int
+~~END~~
+
+~~START~~
+int#!#int
+~~END~~
+
+SELECT current_setting('plan_cache_mode') as plan_mode_after_call
+go
+~~START~~
+text
+force_generic_plan
+~~END~~
+
+
+
+select set_config('plan_cache_mode', 'auto', false)
+go
+~~START~~
+text
+auto
+~~END~~
+
+set babelfish_showplan_all on
+go
+-- cause exception in procedure:
+exec    p_recomp_72 0 with recompile    -- sets plan_cache_mode=force_generic_plan, divides by zero
+go
+~~ERROR (Code: 8134)~~
+
+~~ERROR (Message: division by zero)~~
```

Comment:
This is not a proper test because `showplan_all` does not execute anything inside. However, it seems like it can detect division by zero even without executing it. The test should throw an exception during run-time instead of compile-time.

Reply:
removed showplan